flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/StatementSet.java

```diff
@@ -38,6 +38,29 @@
     /** add {@link Table} with the given sink table name to the set. */
     StatementSet addInsert(String targetPath, Table table, boolean overwrite);
 
+    /**
+     * Adds a statement to insert {@param table} into a table defined by {@param descriptor}.
```

Comment:
```
Adds a statement that the pipeline defined by the given {@link Table} object should be written to a
 table (backed by a {@link DynamicTableSink}) and expressed via the given {@link TableDescriptor}.
```

Somehow we should align `executeInsert` and `addInsert`. `StatementSet` is poorly documented right now.